build/config/rmi_pacta_2023q4_general.json

```diff
@@ -1,5 +1,5 @@
 {
-  "data_share_path": "2023Q4_20240424T120055Z",
+  "data_share_path": "2023Q4_20240709T111731Z",
```

Comment:
side note: `data_share_path` is not a very intuitive name for the PACTA input directory